core/src/mos6502.cpp

```diff
@@ -261,6 +261,23 @@ Pipeline Mos6502::parse_next_instruction() {
     case Instruction::NOP:
         result.push([]() { /* Do nothing. */ });
         break;
+    case Instruction::INC: {
+        if (opcode.address_mode == AddressMode::Zeropage) {
+            result.append(create_zeropage_addressing_steps());
+            result.push([=]() { tmp_ = mmu_->read_byte(effective_address_); });
+            result.push(
+                    [=]() { mmu_->write_byte(effective_address_, tmp_++); });
+            result.push([=]() {
+                set_zero(tmp_);
+                set_negative(tmp_);
+                mmu_->write_byte(effective_address_, tmp_);
+            });
+            break;
+        }
+        std::stringstream err;
```

Comment:
Should this go into a helper function? The same exception with the same error message is also thrown if the instruction is invalid, and should probably also be thrown in some other cases?

Reply:
This isn't needed after merging master, so I deleted it. :D